docs/docusaurus/docs/core/introduction/try_gx.md

```diff
@@ -1,7 +1,153 @@
 ---
 title: Try Great Expectations
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-import InProgress from '../_core_components/_in_progress.md'
+import GxData from '../_core_components/_data.jsx';
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import ReleaseVersionBox from '../../components/versions/_gx_version_code_box.mdx'
+import GxCloudAdvert from '/static/docs/_static_components/_gx_cloud_advert.md'
+
+Follow this guide to install {GxData.product_name}, connect to sample data, build your first Expectation, validate the sample data, and review the Validation Results.  This is an ideal place to start if you're new to GX and would like to experiment with some basic features of GX to see what it has to offer.
+
+## Prerequisites
+
+- <PrereqPythonInstalled/>
+
+## Setup
+
+{GxData.product_name} is a Python library and can be installed using Python's included `pip` tool.
+
+More comprehensive guidance on setting up a Python environment, installing GX, and installing additional dependencies for working with specific data formats and storage environments is available in the [Set up a GX environment](/core/installation_and_setup/install_gx.md) section of the docs. 
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Install the GX 1.0 library.
+
+   The following command will install GX in your Python environment:
+
+   ```bash title="Terminal input"
+   pip install great_expectations
+   ```
+
+   Since this guide provides sample `.csv` data to work with, no additional dependencies need installation.
+
+2. Verify that GX installed successfully.
```

Comment:
```suggestion
2. Verify that {GxData.product_name} installed successfully.
```
How? What does a user need to do?

Reply:
Looks like this is a functional duplicate of 3, probably from when I was copying a portion of the procedure from the "Install GX" guide.  I'll remove the whole thing and re-number these steps.